OxfordBodleian/Juel-Jensen/BDLaethg43.xml

```diff
@@ -238,21 +238,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                                <date notBefore="1800" notAfter="2006"/>
-                                <desc>Relatively regular hand. Broad and slightly cursive characters. The word separator at the end of a line is
-                                occasionally omitted or placed at the beginning of the next line. The ligature
-                                <foreign xml:lang="gez"><hi rend="ligature">ግዚ</hi></foreign> is used.</desc>
-                                <!-- The characters are broad and slightly cursive.  -->
+                                <date notBefore="1800" notAfter="2006">Nineteenth–twentieth century.</date>
                                 <seg type="ink">Black, red.</seg>
-                                <!-- Order is ink, date, desc, rubr -->
-                                <seg type="rubrication">Incipits (one or two rubricated lines), name of the owner, one numeral,
-                                    the words <foreign xml:lang="gez">ወይርኃቁ፡</foreign>
-                                    in <ref target="#ms_i4">textual unit IV</ref> and
-                                    <foreign xml:lang="gez">ሰላም፡ ለከ፡</foreign> in <ref target="#ms_i5">textual unit V</ref>.</seg>
-                                <!-- 
-                                    replace commas with ;
-                                    ; the numeral <foreign xml:lang="gez">፲ወ፪</foreign> in <ref target="#ms_i3">textual unit III</ref> -->
-                                <!-- Textual units should be capitalized -->
+                                <desc>Relatively regular hand. The characters are broad and slightly cursive. The word separator at the end of a line is
+                                occasionally omitted or placed at the beginning of the next line. The ligature
+                                <foreign xml:lang="gez"><hi rend="ligature">ግዚ</hi></foreign> is used.</desc>                            
```

Comment:
Now there are at least two different ways of saying that the gzi ligature is used. Could be standardised?

"The letters `<foreign xml:lang="gez">ግ</foreign>` and `<foreign xml:lang="gez">ዚ</foreign>` are written as a ligature." ?

Or (I see we have this formulation several times): "The ligature `<foreign xml:lang="gez">ግዚ</foreign>` is used."

Reply:
the coexistence of the two formulations seems harmless to me. If it is clear that the two formulations do not refer to different phenomena, I would leave both as expressions of the different cataloguers